README.md

```diff
@@ -18,27 +18,18 @@ Customer Order Cancellation Plugin that allows canceling the unpaid order straig
 
 ## Installation
 
-1. Require plugin using Composer:
- 
-    `composer require sylius/customer-order-cancellation-plugin`.
-
-2. Add plugin class to your `AppKernel`:
-
-    ```php
-    $bundles = [
-       new \Sylius\CustomerOrderCancellationPlugin\SyliusCustomerOrderCancellationPlugin(),
-    ];
-    ```
-
-3. Import routing:
-
-    ```yaml
-    sylius_customer_order_cancellation_plugin:
-        resource: "@SyliusCustomerOrderCancellationPlugin/Resources/config/routing.yml"
-        prefix: /{_locale}
-        requirements:
-            _locale: ^[a-z]{2}(?:_[A-Z]{2})?$
-    ```
+#### Beware!
+
+> This installation instruction assumes that you're using Symfony Flex. If you don't, take a look at the
+[legacy installation instruction](docs/legacy_installation.md). However, we strongly encourage you to use
+Symfony Flex, it's much quicker! :)
+
+To install plugin, just require it with composer:
+
+```bash
+composer require sylius/customer-order-cancellation-plugin
+```
```

Comment:
I'd also add this comment present in RefundPlugin's readme

```
Remember to allow community recipes with composer config extra.symfony.allow-contrib true or during plugin installation process
```

Reply:
done

Reply:
Isn't this also true for https://github.com/Sylius/AdminOrderCreationPlugin/pull/96?

Reply:
and also done 😄